contents/docs/api/post-only-endpoints.mdx

```diff
@@ -114,9 +114,10 @@ Updates user information. Read more in our [identify docs](/docs/product-analyti
 curl -v -L --header "Content-Type: application/json" -d '{
   "api_key": "<ph_project_api_key>",
   "timestamp": "2020-08-16 09:03:11.913767",
-  "context": {},
```

Comment:
`context` is a typo I guess? or does it exist?

Reply:
Probably an old thing